typescript/libs/global_vars.py

```diff
@@ -34,7 +34,9 @@
 def get_node_path():
     return _node_path
 
-TSC_PATH = os.path.join(PLUGIN_DIR, "tsserver", "tsc.js")
+_tsc_path = None
```

Comment:
There's a comment above that says where `_node_path` gets initialized. Can you add a similar one here?